src/components/mfa-reset-with-ipv/mfa-reset-with-ipv-controller.ts

```diff
@@ -0,0 +1,30 @@
+import { MfaResetAuthorizeInterface } from "./types";
+import { mfaResetAuthorizeService } from "./mfa-reset-authorize-service";
+import { ExpressRouteFunc } from "../../types";
+import { Request, Response } from "express";
+import { BadRequestError } from "../../utils/error";
+
+export function mfaResetWithIpvGet(
+  service: MfaResetAuthorizeInterface = mfaResetAuthorizeService()
+): ExpressRouteFunc {
+  return async function (req: Request, res: Response) {
+    const { email } = req.session.user;
+    const { sessionId, clientSessionId, persistentSessionId } = res.locals;
+
+    const result = await service.ipvRedirectUrl(
+      sessionId,
+      clientSessionId,
+      persistentSessionId,
+      req,
+      email
+    );
+
+    const ipvCoreURL = result.data.authorize_url;
```

Comment:
Should this come after the below `if(!result.success)`? I imagine we can't assume an authorize_url if the response is not a success response?